install_config/aggregate_logging.adoc

```diff
@@ -473,6 +473,62 @@ $ oc delete secret logging-fluentd logging-elasticsearch \
     logging-kibana-ops-proxy
 ----
 
+
+== Upgrading the EFK Stack
+
```

Comment:
Can we call out specifically that this is 3.1 -> 3.2?  Mostly as a place holder -- when it comes to it, 3.2 -> 3.3 this will be either more complicated or automated